packages/framework-integration-tests/integration/provider-specific/deploy.ts

```diff
@@ -1,5 +1,6 @@
 import * as path from 'path'
-import { runCommand } from '../../helper/run-command'
```

Comment:
Would you mind calling this file `cli-helper.ts` and put it inside the `integration/helper` folder?

I think it makes more sense.